x/cellarfees/keeper/keeper_test.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/peggyjv/sommelier/v4/x/cellarfees/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGettingSettingFeeAccrualCounters(t *testing.T) {
+	env := CreateTestEnv(t)
```

Comment:
With the merging of the mock framework and its usage in auction as well, we may want to consider moving the unit tests to that format.